### PR TITLE
K8s devcontainer

### PR DESCRIPTION
**Pull request by SeanCheatham**
## Purpose
- Enables running a testnet simulation from GitHub Codespaces
## Approach
- Modify .devcontainer to install k8s, Scala, helm, and other dependencies
## Testing
- Launched local testnet
  image
## Tickets
- BN-864

### CODE REVIEW

**Comment by hwbehrens** [comment]
Based on the conflict with PR #2827, I think it makes sense to merge this one first, rework that one, and then merge that one to the new repo. @admiraladmirable what do you think?

**Comment by admiraladmirable** [comment]
> Based on the conflict with PR #2827, I think it makes sense to merge this one first, rework that one, and then merge that one to the new repo. @admiraladmirable what do you think?

The new workflow is this:

```
helm repo add topl https://topl.github.io/helm-charts
helm repo update

helm upgrade --install \
    $DEMO_NAME
    topl/bifrost-consensus-testnet \
    --namespace $DEMO_NAME \
    --create-namespace 
```

If testing is too difficult since it's towards the end of the sprint, we can hold off on merging my other PR which removes the local helm chart, but I tested this locally and it seemed to work. I will leave the decision up to @SeanCheatham

File: docs/DevContainerSimulatorGuide.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Instructions for running a testnet simulation in a Devcontainer environment (i.e. GitHub Codespaces).
+
+## Installation
+All pre-requisites should be automatically installed, including: Java, SBT, Docker, minikube, kubectl, helm, and k9s
+
+### Bifrost
+1. Run `sbt Docker/publishLocal` from the command line.
+1. Run `./docs/scripts/docker_tag_minikube.sh` from the command line.
+
+### Helm Charts
```
Comment (SeanCheatham) [inline docs/DevContainerSimulatorGuide.md:11]: @hwbehrens @admiraladmirable Just following up on the conversation on the PR:

These steps here should hopefully reflect the changes Rick made to the location of the helm charts.  I think Rick's PR can be merged now.  I'll just need to make a few cleanup changes on this branch after it's merged.

Comment (admiraladmirable) [reply docs/DevContainerSimulatorGuide.md:11]: I think this is a suitable workaround to use the override `value.yaml` file in the new helm repo.

The production way to install is in my comment above, but I think your workaround makes sense in this context.

**Pull request merged**